webapp/src/language/es.json

```diff
@@ -94,7 +94,13 @@
     "community": "Comunidad",
     "changelog": "Registro de cambios",
     "github": "Repositorio de GitHub",
-    "telegram": "Grupo en Telegram"
+    "telegram": "Grupo en Telegram",
+    "lacchain": "Monitor de infraestructura y panel de control de red LACChain EOSIO",
+    "mainnet": "Monitor de infraestructura y panel de control de la red EOS Mainnet",
+    "lacchain>nodes": "Monitor de estado de los nodos LACChain EOSIO",
+    "mainnet>nodes": "Monitor de estado de los nodos productores de bloques de EOS Mainnet",
+    "telos": "Monitor de infraestructura y panel de control de red Telos Testnet",
+    "jungle": "Monitor de infraestructura y panel de control de red Jungle Testnet"
```

Comment:
@tetogomez  
what about the other pages?

- rewards 
- entities 
- geo location 
- etc

Reply:
done